specs/wp-calypso-gutenberg-post-editor-spec.js

```diff
@@ -987,7 +987,8 @@ describe( `[${ host }] Calypso Gutenberg Editor: Posts (${ screenSize })`, funct
 
 		step( 'Can publish and view content', async function() {
 			const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
-			return await gEditorComponent.publish( { visit: true } );
+			await gEditorComponent.publish( { visit: true } );
+			this.skip();
```

Comment:
I am assuming this skip was just for testing.

Reply:
🤦‍♂️ Good guess :) Thanks for catching this!

Reply:
Done.